controllers/storagecluster/cephrbdmirrors.go

```diff
@@ -0,0 +1,158 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+
+	ocsv1 "github.com/openshift/ocs-operator/api/v1"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type ocsCephRbdMirrors struct{}
+
+func (r *StorageClusterReconciler) deleteCephRbdMirrorInstance(cephRbdMirror *cephv1.CephRBDMirror, existing *cephv1.CephRBDMirror) error {
+	var err error
+
+	if cephRbdMirror.GetDeletionTimestamp().IsZero() {
+		r.Log.Info("Deleting CephRbdMirror.", "CephRbdMirror", klog.KRef(cephRbdMirror.Namespace, cephRbdMirror.Name))
+		err = r.Client.Delete(context.TODO(), existing)
+	}
+	return err
+}
+
+// newCephRbdMirrorInstances returns the cephRbdMirror instances that should be created
+// on first run.
+func (r *StorageClusterReconciler) newCephRbdMirrorInstances(initData *ocsv1.StorageCluster) ([]*cephv1.CephRBDMirror, error) {
+
+	var rbdMirroringSpec cephv1.RBDMirroringSpec
+	rbdMirroringSpec.Count = 1
+	rbdMirroringSpec.Resources = corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("1"),
+			corev1.ResourceMemory: resource.MustParse("2Gi"),
+		},
+		Requests: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("1"),
+			corev1.ResourceMemory: resource.MustParse("2Gi"),
+		},
+	}
```

Comment:
This should be added here: https://github.com/openshift/ocs-operator/blob/0b1d4bd26829b15538b71616a463514f2862dc5a/controllers/defaults/resources.go#L11